src/tests/elementary/spec/efl_test_gfx_view.c

```diff
@@ -6,10 +6,12 @@
 #include "efl_ui_spec_suite.h"
 #include "suite_helpers.h"
 
+#ifdef HAVE_ECTOR
+#else
 /* spec-meta-start
-      {"test-interface":"Efl.Gfx.View",
-       "test-widgets": ["Efl.Ui.Vg_Animation"]}
+      {"test-interface":"Efl.Gfx.View"}
    spec-meta-end */
+#endif
```

Comment:
Is this as intended? It seems as something went wrong here, or at least it should be a `#ifndef` instead of a `#ifdef`

Reply:
The `#ifdef` here actually makes no changes at all to the comments. This is a leftover of an attempt to catch that behaviour.